suggested-content/word-2-vec-service.py

```diff
@@ -27,6 +31,14 @@ def documentCheckPhrases():
     response = checkPhrases(grams)
     return jsonify(results = response)
 
+@app.route('/doc', methods=['POST'])
+def similarDocs():
+    doc = request.json["doc"]   
```

Comment:
It's not going to come in json format anymore. Instead it's one line with the url, line break, then the text content and another line break to separate the documents. I'll upload a sample of the file to S3